src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -78,10 +85,80 @@ Globalnet currently relies on `kube-proxy` and thus will only work with deployme
 Connectivity is only part of the solution as pods still need to know the IPs of services on remote clusters.
 
 This is achieved by enhancing [lighthouse](https://github.com/submariner-io/lighthouse) with support for Globalnet. The Lighthouse
-controller adds the service's globalIp to the `ServiceImport` object that is distributed to all clusters. The [lighthouse
-plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the Service's globalIp when replying to DNS
-queries for the Service.
+controller uses service's global IP instead of `clusterIP` when creating `ServiceImport` for services of type `ClusterIP`. In case of
+services of type `Headless`, pod's global IP is used instead of podIp when creating `EndpointSlice` objects to be distributed to other clusters.
+The [lighthouse plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the service's global IP from `ServiceImport`
+when replying to DNS queries for the services of type `ClusterIP`. For headless services, it returns global IPs in the corresponding `EndpointSlices`
+for pods backing the service.
 
 ## Building
 
 Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard Submariner build.
+
+## Usage
+
+Refer to the [Quickstart Guides](../../quickstart/) on how to deploy Submariner with Globalnet enabled. For most deployments users will not
+need to do anything else once deployed. But users can create `GlobalEgressIP`s or edit `ClusterGlobalEgressIP` for specific use cases.
+
+### Ephemeral Port Exhaustion
+
+By default, 8 cluster-scoped global IPs are allocated which allows for `~8x64k` active ephemeral ports. If those are still
+not enough for a cluster, users can increase the number of ClusterGlobalIPs allocated.
+Note that this assumes that there are still some global IPs available in the cluster pool. To increase the number of global IPs allocated
+for `ClusterGlobalEgressIP`s, create the following object with `NumberOfIPs` set to required value.
+
+```yaml
+   apiVersion: submariner.io/v1alpha1
+   kind: ClusterGlobalEgressIP
+   metadata:
+     name: cluster-egress.submariner.io
+   spec:
+     NumberOfIPs: 9
+```
+
+{{% notice note %}}
+Only the `ClusterGlobalEgressIP` resource with the name `cluster-egress.submariner.io` is recognized by Globalnet. This resource is automatically
+created with the default number of IPs.
+{{% /notice %}}
+
+### Global IP for a Namespace
+
+For use cases where user wants all pods in a namespace to use a unique global IP instead of one of the `ClusterGlobalEgressIP`s,
+users can create
+GlobalEgressIP for that namespace.
```

Comment:
```suggestion
If it's desired for all pods in a namespace to use a unique global IP instead of one of the cluster-scoped IPs, a user can create a `GlobalEgressIP` resource in that namespace:
```